src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -21,8 +21,9 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       x: {
         type: 'timeseries',
         tick: {
+          culling: false,
```

Comment:
What is this?

Reply:
it's for show all dates in x axis
![image](https://user-images.githubusercontent.com/8861133/64428600-81028100-d08a-11e9-9ac6-c9a18897093b.png)

Reply:
👍